code/controllers/subsystem/jobs.dm

```diff
@@ -351,6 +351,7 @@ var/datum/subsystem/job/SSjob
 			player.ready = 0
 			unassigned -= player
 			ticker.mode.antag_candidates -= player.mind
+			to_chat(player, "<b><font color='red'>You were returned to the lobby because your preference unavailable.</font></b>")
```

Comment:
```suggestion
			to_chat(player, "<span class='alert bold'>You were returned to the lobby because your job preferences unavailable. You can change this behavior in preferences.</font>")
```

Reply:
На <span class='alert bold'> ругается Travis, поэтому я color='red' и ставил

Reply:
Надо не мудрить новые какие-то стили, а смотреть в таблицу стилей для гунчата, если нужен красный и болд, есть boldannounce.